### PR TITLE
Fix bug in admin change scope page (also allow admins to change the scope for any year, not just current year)

### PR DESCRIPTION
**Pull request by jamesgriff**

### CODE REVIEW

File: GenderPayGap.WebUI/Controllers/Admin/AdminOrganisationScopeController.cs [hunk 1]
```diff
@@ -103,18 +110,19 @@ public async Task<IActionResult> ChangeScopePost(long id, AdminChangeScopeViewMo
...
         }
 
-        private void RetireOldScopesForCurrentSnapshotDate(Organisation organisation)
+        private void RetireOldScopesForCurrentSnapshotDate(Organisation organisation, int year)
         {
-            IOrderedEnumerable<OrganisationScope> organisationScopesForCurrentSnapshotDate = organisation.OrganisationScopes
-                .OrderByDescending(s => s.SnapshotDate);   
+            IEnumerable<OrganisationScope> organisationScopesForCurrentSnapshotDate =
+                organisation.OrganisationScopes
+                .Where(scope => scope.SnapshotDate.Year == year);
```
Comment (jamesgriff) [inline GenderPayGap.WebUI/Controllers/Admin/AdminOrganisationScopeController.cs]: This was the bug - we were setting all scope (for all years) to Retired.
Now we're only setting all scopes for the current year to Retired

Comment (raamSoftwire) [reply GenderPayGap.WebUI/Controllers/Admin/AdminOrganisationScopeController.cs]: Good spot. Thanks for fixing this

File: GenderPayGap.WebUI/Views/Admin/ViewOrganisation.cshtml [hunk 2]
```diff
@@ -179,10 +179,19 @@
...
-                @(Model.LatestScope.ScopeStatus)
+                @{
+                    var scopes = Model.OrganisationScopes
+                        .Where(s => s.Status == ScopeRowStatuses.Active)
+                        .OrderByDescending(s => s.SnapshotDate);
+                }
+                @foreach (var scope in scopes)
+                {
+                    <b>
+                        @($"{scope.SnapshotDate.Year}-{(scope.SnapshotDate.Year + 1) % 100}"):
```
Comment (raamSoftwire) [inline GenderPayGap.WebUI/Views/Admin/ViewOrganisation.cshtml]: This is neat, but took me a second to figure out what is going on. Could we add a comment, or pull out a method for this formatting with a nice, descriptive method name?

Comment (raamSoftwire) [reply GenderPayGap.WebUI/Views/Admin/ViewOrganisation.cshtml]: I looks like we would re-use it a few times, so maybe a method over a comment makes sense?

**Pull request merged**